packages/react/src/ReactDebugCurrentFrame.js

```diff
@@ -7,18 +7,46 @@
  * @flow
  */
 
+import type {ReactElement} from 'shared/ReactElementType';
+
+import describeComponentFrame from 'shared/describeComponentFrame';
+import getComponentName from 'shared/getComponentName';
+
 const ReactDebugCurrentFrame = {};
 
+let currentlyValidatingElement = (null: null | ReactElement);
+
+export function setCurrentlyValidatingElement(element: null | ReactElement) {
```

Comment:
I didn't put it on the `ReactDebugCurrentFrame` object because I don't want it to be accessible by the renderers.